scripts/buildwin.bat

```diff
@@ -0,0 +1,20 @@
+
+REM Use the downloaded OpenSSL, for all other libraries the bundled version is used.
+REM On Appveyor different OpenSSL versions are available, therefore the directory contains the version information.
+set BORG_OPENSSL_PREFIX=C:\OpenSSL-v111-Win64
+set BORG_USE_BUNDLED_B2=YES
+set BORG_USE_BUNDLED_LZ4=YES
+set BORG_USE_BUNDLED_ZSTD=YES
+set BORG_USE_BUNDLED_XXHASH=YES
+
+REM Somehow on my machine rc.exe was not found. Adding the Windows Kit to the path worked.
+set PATH=%PATH%;C:\Program Files (x86)\Windows Kits\10\bin\10.0.18362.0\x64
+
+REM Run the build in the project directory.
+SET WORKPATH=%~dp0\..
+pushd %WORKPATH%
```

Comment:
why did you add this? guess this is specific to your machine and does not work for other developers.

you can just have the toplevel dir of the repo as current directory and run the script using `scripts\buildwin[.bat]`.

Reply:
If this is removed, you **must** run it from the repo directory. With this code, the script can be called independent of the current working directory.

Reply:
sorry, i didn't really understand what it means, i was assuming ~ works similar as on linux...

so, keep it if it is useful.

Reply:
Yeah, strange things in batch files... `%~dp0` actually means **d**rive letter and **p**ath of first argument which is passed to the script, which is the full path of the script itself.